idseq_dag/steps/download_accessions.py

```diff
@@ -180,7 +179,7 @@ def fetch_sequence_for_thread(error_flags, accession, accession_out_file,
                         else:
                             msg = f"All retries failed for getting sequence by accession ID {accession}: {e}"
                         raise RuntimeError(msg)
-        except:
+        except Exception:
```

Comment:
Similar to my earlier comments, do not do this.   Can we perhaps just disable the linter rule in the config file?